[2023-08-22 16:19:24,346] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,346] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道新建通道
[2023-08-22 16:19:24,346] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,346] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,346] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,346] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道9729', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,347] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,378] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}]
[2023-08-22 16:19:24,378] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 44, 'parentId': None, 'sort': 21, 'name': '通道9729', 'description': '', 'type': 1, 'typeTag': 'TCP', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'enable': True, 'createTime': '2023-08-22T16:19:24.3717421+08:00', 'updateTime': '2023-08-22T16:19:24.3717428+08:00'}
[2023-08-22 16:19:24,378] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-22 16:19:24,378] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-22 16:19:24,383] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,383] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道通道名称为空
[2023-08-22 16:19:24,383] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,383] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,383] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,383] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道9990', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,383] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,403] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}]
[2023-08-22 16:19:24,403] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 45, 'parentId': None, 'sort': 22, 'name': '通道9990', 'description': '', 'type': 1, 'typeTag': 'TCP', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'enable': True, 'createTime': '2023-08-22T16:19:24.3971872+08:00', 'updateTime': '2023-08-22T16:19:24.3971879+08:00'}
[2023-08-22 16:19:24,403] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-22 16:19:24,403] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-22 16:19:24,408] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,408] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道设备IP为空
[2023-08-22 16:19:24,408] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,408] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,408] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,408] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道2529', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': 'null', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,409] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,416] logger_util.py->error_log line:65 [INFO] 断言失败:status_code不等于200
[2023-08-22 16:19:24,416] logger_util.py->write_log line:61 [INFO] 接口请求失败！
[2023-08-22 16:19:24,416] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-08-22 16:19:24,417] logger_util.py->error_log line:65 [INFO] 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

[2023-08-22 16:19:24,417] logger_util.py->error_log line:65 [INFO] 分析YAML文件异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 249, in analysis_yaml
    self.validate_result(yq_result, return_text, status_code)
  File "E:\Project_python\api_cms\common\requests_util.py", line 401, in validate_result
    error_log("断言异常：异常信息: %s" % str(traceback.format_exc()))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200


[2023-08-22 16:19:24,521] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,521] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道设备IP非法
[2023-08-22 16:19:24,521] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,521] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,521] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,521] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道2704', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': 'abc', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,521] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,527] logger_util.py->error_log line:65 [INFO] 断言失败:status_code不等于200
[2023-08-22 16:19:24,527] logger_util.py->write_log line:61 [INFO] 接口请求失败！
[2023-08-22 16:19:24,527] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-08-22 16:19:24,527] logger_util.py->error_log line:65 [INFO] 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

[2023-08-22 16:19:24,528] logger_util.py->error_log line:65 [INFO] 分析YAML文件异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 249, in analysis_yaml
    self.validate_result(yq_result, return_text, status_code)
  File "E:\Project_python\api_cms\common\requests_util.py", line 401, in validate_result
    error_log("断言异常：异常信息: %s" % str(traceback.format_exc()))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200


[2023-08-22 16:19:24,555] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,555] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道设备ID为空
[2023-08-22 16:19:24,555] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,555] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,555] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,555] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道9935', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': 'null', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,555] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,561] logger_util.py->error_log line:65 [INFO] 断言失败:status_code不等于200
[2023-08-22 16:19:24,561] logger_util.py->write_log line:61 [INFO] 接口请求失败！
[2023-08-22 16:19:24,561] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-08-22 16:19:24,561] logger_util.py->error_log line:65 [INFO] 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

[2023-08-22 16:19:24,561] logger_util.py->error_log line:65 [INFO] 分析YAML文件异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 249, in analysis_yaml
    self.validate_result(yq_result, return_text, status_code)
  File "E:\Project_python\api_cms\common\requests_util.py", line 401, in validate_result
    error_log("断言异常：异常信息: %s" % str(traceback.format_exc()))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200


[2023-08-22 16:19:24,588] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,588] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道设备ID非法
[2023-08-22 16:19:24,588] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,588] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,588] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,588] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道4413', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': 'abc', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,589] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,594] logger_util.py->error_log line:65 [INFO] 断言失败:status_code不等于200
[2023-08-22 16:19:24,594] logger_util.py->write_log line:61 [INFO] 接口请求失败！
[2023-08-22 16:19:24,594] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-08-22 16:19:24,594] logger_util.py->error_log line:65 [INFO] 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

[2023-08-22 16:19:24,595] logger_util.py->error_log line:65 [INFO] 分析YAML文件异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 249, in analysis_yaml
    self.validate_result(yq_result, return_text, status_code)
  File "E:\Project_python\api_cms\common\requests_util.py", line 401, in validate_result
    error_log("断言异常：异常信息: %s" % str(traceback.format_exc()))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200


[2023-08-22 16:19:24,621] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,621] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道端口号为空
[2023-08-22 16:19:24,621] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,621] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,621] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,622] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道9649', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': 'null', 'HeadAddress': '1', 'ConnectTimeOut': 'null', 'ResponseTimeout': 'null', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': 'null'}}, 'copyConfig': None}
[2023-08-22 16:19:24,622] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,627] logger_util.py->error_log line:65 [INFO] 断言失败:status_code不等于200
[2023-08-22 16:19:24,627] logger_util.py->write_log line:61 [INFO] 接口请求失败！
[2023-08-22 16:19:24,627] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-08-22 16:19:24,628] logger_util.py->error_log line:65 [INFO] 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

[2023-08-22 16:19:24,628] logger_util.py->error_log line:65 [INFO] 分析YAML文件异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 249, in analysis_yaml
    self.validate_result(yq_result, return_text, status_code)
  File "E:\Project_python\api_cms\common\requests_util.py", line 401, in validate_result
    error_log("断言异常：异常信息: %s" % str(traceback.format_exc()))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200


[2023-08-22 16:19:24,654] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,654] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道端口号非法
[2023-08-22 16:19:24,654] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,654] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,654] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,654] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道7996', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': 'abc', 'HeadAddress': '1', 'ConnectTimeOut': 'abc', 'ResponseTimeout': 'abc', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': 'abc'}}, 'copyConfig': None}
[2023-08-22 16:19:24,654] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,660] logger_util.py->error_log line:65 [INFO] 断言失败:status_code不等于200
[2023-08-22 16:19:24,660] logger_util.py->write_log line:61 [INFO] 接口请求失败！
[2023-08-22 16:19:24,660] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-08-22 16:19:24,660] logger_util.py->error_log line:65 [INFO] 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

[2023-08-22 16:19:24,661] logger_util.py->error_log line:65 [INFO] 分析YAML文件异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 249, in analysis_yaml
    self.validate_result(yq_result, return_text, status_code)
  File "E:\Project_python\api_cms\common\requests_util.py", line 401, in validate_result
    error_log("断言异常：异常信息: %s" % str(traceback.format_exc()))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 347, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200


[2023-08-22 16:19:24,686] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,687] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道连接超时为空
[2023-08-22 16:19:24,687] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,687] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,687] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,687] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道7127', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,687] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,707] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}]
[2023-08-22 16:19:24,707] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 46, 'parentId': None, 'sort': 23, 'name': '通道7127', 'description': '', 'type': 1, 'typeTag': 'TCP', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'enable': True, 'createTime': '2023-08-22T16:19:24.7012751+08:00', 'updateTime': '2023-08-22T16:19:24.7012757+08:00'}
[2023-08-22 16:19:24,707] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-22 16:19:24,707] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-22 16:19:24,712] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,712] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道连接超时非法
[2023-08-22 16:19:24,712] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,712] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,712] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,712] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道8144', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,712] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,731] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}]
[2023-08-22 16:19:24,731] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 47, 'parentId': None, 'sort': 24, 'name': '通道8144', 'description': '', 'type': 1, 'typeTag': 'TCP', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'enable': True, 'createTime': '2023-08-22T16:19:24.7254557+08:00', 'updateTime': '2023-08-22T16:19:24.7254562+08:00'}
[2023-08-22 16:19:24,731] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-22 16:19:24,731] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-22 16:19:24,736] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,736] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道响应超时为空
[2023-08-22 16:19:24,736] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,736] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,736] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,736] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道6642', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,736] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,756] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}]
[2023-08-22 16:19:24,757] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 48, 'parentId': None, 'sort': 25, 'name': '通道6642', 'description': '', 'type': 1, 'typeTag': 'TCP', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'enable': True, 'createTime': '2023-08-22T16:19:24.7508133+08:00', 'updateTime': '2023-08-22T16:19:24.7508139+08:00'}
[2023-08-22 16:19:24,757] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-22 16:19:24,757] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-22 16:19:24,761] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,762] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道响应超时非法
[2023-08-22 16:19:24,762] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,762] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,762] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,762] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道5826', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,762] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,782] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}]
[2023-08-22 16:19:24,782] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 49, 'parentId': None, 'sort': 26, 'name': '通道5826', 'description': '', 'type': 1, 'typeTag': 'TCP', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'enable': True, 'createTime': '2023-08-22T16:19:24.7763791+08:00', 'updateTime': '2023-08-22T16:19:24.7763798+08:00'}
[2023-08-22 16:19:24,782] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-22 16:19:24,783] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-22 16:19:24,787] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,787] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道PDU长度为空
[2023-08-22 16:19:24,787] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,788] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,788] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,788] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道6705', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,788] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,810] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}]
[2023-08-22 16:19:24,810] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 50, 'parentId': None, 'sort': 27, 'name': '通道6705', 'description': '', 'type': 1, 'typeTag': 'TCP', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'enable': True, 'createTime': '2023-08-22T16:19:24.8025119+08:00', 'updateTime': '2023-08-22T16:19:24.8025127+08:00'}
[2023-08-22 16:19:24,810] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-22 16:19:24,810] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-22 16:19:24,814] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-22 16:19:24,814] logger_util.py->write_log line:61 [INFO] 接口名称：创建通道PDU长度非法
[2023-08-22 16:19:24,814] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-22 16:19:24,814] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-22 16:19:24,814] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-22 16:19:24,814] logger_util.py->write_log line:61 [INFO] 请求参数：{'parentId': None, 'type': 1, 'name': '通道6982', 'description': '', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'copyConfig': None}
[2023-08-22 16:19:24,814] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-22 16:19:24,834] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}]
[2023-08-22 16:19:24,834] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 51, 'parentId': None, 'sort': 28, 'name': '通道6982', 'description': '', 'type': 1, 'typeTag': 'TCP', 'config': {'suppiler': 'ModBus', 'model': 'Modbus_TCP', 'protocol': 'Modbus_TCP', 'parameters': {'ServerName': '127.0.0.1', 'DeviceID': '1', 'Port': '502', 'HeadAddress': '1', 'ConnectTimeOut': '502', 'ResponseTimeout': '502', 'Int32ByteOrder': '2143', 'Int64ByteOrder': '2143', 'UInt32ByteOrder': '2143', 'UInt64ByteOrder': '2143', 'F32ByteOrder': '3412', 'F64ByteOrder': '2143', 'StrByteOrder': '2143', 'EnableIndependentLink': '禁用', 'EnablePduDiagnosis': '启用', 'PduLength': '502'}}, 'enable': True, 'createTime': '2023-08-22T16:19:24.8289812+08:00', 'updateTime': '2023-08-22T16:19:24.8289816+08:00'}
[2023-08-22 16:19:24,834] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-22 16:19:24,834] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

